overlord/snapstate/snapstate.go

```diff
@@ -52,9 +52,14 @@ const (
 	// for use in the interim time while we have the backward compatible
 	// support
 	firstInterimUsableFlagValue
-	// if we need flags for just SnapSetup it may be easier
-	// to start a new sequence from the other end with:
-	// 0x40000000 >> iota
+)
+
+// SnapSetupFlags are flags stored in SnapSetup to control snap manager tasks.
+type SnapSetupFlags Flags
+
+const (
+	// RollbackOp means the given snap got rolled back
+	RollbackOp = 0x40000000 >> iota
```

Comment:
sorry, probably was confusing but I also meant:

RollbackOp SnapSetupFlags = 0x40000000 >> iota

so it cannot be mixed easily with SnapStateFlags